src/Makefile

```diff
@@ -315,12 +318,12 @@ LIBS+=$(PNG_LDFLAGS)
 CFLAGS+=$(PNG_CFLAGS)
 endif
 
-ifdef HAVE_LIBGME
+ifndef NOGME
 OPTS+=-DHAVE_LIBGME
 
 LIBGME_PKGCONFIG?=libgme
 LIBGME_CFLAGS?=$(shell $(PKG_CONFIG) $(LIBGME_PKGCONFIG) --cflags)
-LIBGME_LDFLAGS?=$(shell $(PKG_CONFIG) $(LIBGME_PKGCONFIG) --libs)
```

Comment:
never, the old libpng12 used to do, but libpng16 does not anymore

use  pkg-config on zlib to get the cflags and libs flags